src/database/schema/ChatSchema.ts

```diff
@@ -280,15 +281,26 @@ class ChatSchema implements BaseDbSchema {
     channelId: string,
     message: string,
     localDb: SQLiteDatabase,
-    type: 'regular' | 'system' = 'regular',
-    status: 'pending' | 'sent' = 'pending'
+    type: 'regular' | 'system' | 'reply' = 'regular',
+    status: 'pending' | 'sent' = 'pending',
+    json: string | null = null
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function signature has been updated to include a new parameter `json`. This change seems fine as long as all calls to this function have been updated accordingly.
```diff
-    type: 'regular' | 'system' = 'regular',
-    status: 'pending' | 'sent' = 'pending'
+    type: 'regular' | 'system' | 'reply' = 'regular',
+    status: 'pending' | 'sent' = 'pending',
+    json: string | null = null
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->